modules/core/src/main/java/org/apache/ignite/dump/DumpReader.java

```diff
@@ -101,6 +101,9 @@ public DumpReader(DumpReaderConfiguration cfg, IgniteLogger log) {
                     }
                 }
 
+                Map<String, Boolean> compressions = dump.metadata().stream()
```

Comment:
I suppose the compression value should be the same for all cache groups within a dump?